doc/source/dev/missing-bits.rst

```diff
@@ -24,6 +24,11 @@ rule.
 
 Coding Style and Guidelines
 ---------------------------
+Note that docstrings should be generally made up of ASCII characters
+in spite of being Unicode. Additionally, lower case greek letters
+"αβγδεζηθικλμνξoπρστυϕχψω" as well the following upper case greek letters
+"ΓΔΘΛΞΠΣϒΦΨΩ" are allowed. Also the following symbols may be used:
+"®ő∫≠≥≤±∞²³".
```

Comment:
Since this is developer documentation, shall we refer the developer to `tools/unicode-check.py` rather than having to keep this (in a separate file) up to date? Or can we import the list from `unicode-check.py` or a common location?